x/millions/types/errors.go

```diff
@@ -36,6 +36,8 @@ var (
 	ErrInvalidDepositState               = errorsmod.Register(ModuleName, 1208, "Deposit is in an invalid state")
 	ErrInvalidPoolAccountAddress         = errorsmod.Register(ModuleName, 1209, "Invalid pool module account address")
 	ErrInvalidRestorerAddress            = errorsmod.Register(ModuleName, 1210, "Invalid restorer address")
+	ErrInvalidAutoCompoundSponsorCombo   = errorsmod.Register(ModuleName, 1211, "Only auto compound operations can sponsor a deposit")
```

Comment:
The error message should be more clear "...sponsor a deposit through claim process"